src/builder.rs

```diff
@@ -12,6 +12,8 @@ pub struct SessionBuilder {
     port: Option<String>,
     keyfile: Option<std::path::PathBuf>,
     connect_timeout: Option<String>,
+    server_alive_interval: Option<String>,
+    server_alive_count_max: Option<String>,
```

Comment:
These can be stored as integers, since we need to use `format!` when passing them anyway.